web/layouts/partials/package-details.html

```diff
@@ -0,0 +1,111 @@
+<h3>{{ .name }} {{ .version }}</h3>
+<hr/>
+<table class="details">
+  <tr>
+    <th>Architecture:</th>
+    <td>{{ .arch }}</td>
+  </tr>
+  <tr>
+    <th>Base Package:</th>
+    <td>
+      {{ .base }}
+      (<a href="https://github.com/osam-cologne/archlinux-proaudio/blob/master/packages/{{ .base }}/PKGBUILD" target="_blank">PKGBUILD</a>)
+    </td>
+  </tr>
+  <tr>
+    <th>Description:</th>
+    <td>{{ .desc }}</td>
+  </tr>
+  {{ if .url }}
+  <tr>
+    <th>Upstream URL:</th>
+    <td><a href="{{ .url }}" target="_blank">{{ .url }}</a></td>
+  </tr>
+  {{ end }}
+  {{ if .groups }}
+  <tr>
+    <th>Groups:</th>
+    <td>{{ range .groups }}{{ . }}<br/>{{ end }}</td>
```

Comment:
Would it be possible to sort groups alphabetically here?

Also for depends, makedepends, provides, conflicts, replces, etc.

Reply:
done: 4b6845b5d2e46b5a917a2b8c060feef9cfd689d0

Reply:
(the baseURL doesn't fully work with the /hugo-test path but it will work after merging)